packages/docs/concepts/blocks.yaml

```diff
@@ -24,56 +24,35 @@ _ref:
-              - Validation is invoked using the [`Validate`](/Validate) action.
-              ##### Block events
-              - All blocks have`onMount` and `onMountAsync` events.
-              - Each block implements it's own additional events such as `onClick` etc.
-              ##### Block loading
-              - Some blocks gracefully handle a loading state while `onMount` events are being executed.
-              - A block's default loading can be overwritten by defining custom `skeleton` settings on a block.
-
-            -------
-            A Lowdefy page is compiled out of an arrangement of blocks. Every HTML element of this page is rendered as a result of a block placed and configured on the page. Blocks make it simple for Lowdefy developers to create apps since they only have to decide what block type to use, where in the layout the block should render, and what the block should do by defining the block's `properties`. How a block implements these `properties` is up to the specific block type selected.
+            A Lowdefy page is compiled out of an arrangement of blocks. Every HTML element of this page is rendered as a result of a block placed and configured on the page. Blocks make it simple for Lowdefy developers to create apps since they only have to decide which block type to use, where in the layout the block should render, and what the block should do by defining the block's `properties`. How a block implements these `properties` is up to the specific block type selected.
 
             Lowdefy offers a list of over 30 block types to cater for a wide range of use cases. All blocks are categorized according to their primary function:
             - `display` - Display page elements.
             - `input` - Modify a value in [`state`](/context-and-state).
-            - `context` - Create a new [`context`](/context-and-state).
             - `container` - Render other blocks into [`content areas`](/layout).
             - `list` - Render `content areas` and blocks for each element in the data array.
 
-            When `state` updates or a requests call completes, the Lowdefy engine reevaluates all operators and rerenders blocks for which the operator evaluation is different from the previous render result. The result is _live updates_  to all blocks on a page. Operators can be used to build _live update_ logic into all block fields, except for the `id`, `type`, `areas`, `blocks` and `loading` fields.
-
-            # Block Schema
+            When `state` updates or a requests call completes, the Lowdefy engine reevaluates all operators and rerenders blocks for which the operator evaluation is different from the previous render result. The result is _live updates_ to all blocks on a page. Operators can be used to build _live update_ logic into all block fields, except for the `id`, `type`, `areas`, `blocks` and `loading` fields.
```

Comment:
The `loading` field should be removed from this statement.

Reply:
Change made in b4bcd68767166479063cf5cf27a7b242846eae2c